app/controllers/home_controller.rb

```diff
@@ -0,0 +1,5 @@
+class HomeController < ApplicationController
+    def index
```

Comment:
Do you want this for something? Tech. w/ rails magic you don't need it...

Reply:
We had it there for clarity. But we will get rid of it since we don't need it! Thank you!